tracing-opentelemetry/src/layer.rs

```diff
@@ -364,11 +365,21 @@ where
     {
         OpenTelemetryLayer {
             tracer,
+            span_timings_enabled: self.span_timings_enabled,
             get_context: WithContext(OpenTelemetryLayer::<S, Tracer>::get_context),
             _registry: self._registry,
         }
     }
 
+    /// Sets whether or not spans metadata should include the _busy time_ (total time for which it was entered),
+    /// and _idle time_ (total time the span existed but was not entered).
+    pub fn with_span_timings(self, span_timings_enabled: bool) -> Self {
```

Comment:
May want the name to distinguish between the span timings that are currently tracked and the new timings you are introducing? something like `with_tracked_inactivity` or other description of what these timings are.